rust/src/host_functions.rs

```diff
@@ -48,9 +48,9 @@ pub mod host_functions_impl {
             buf
         }
 
-        fn sha3_512(message: &[u8]) -> [u8; 64] {
```

Comment:
This is being removed because it's not used?

Reply:
That’s right. As far as I could check in projects around `ibc-rs`, they are not using it, and there isn't any `HashOp` variant already for it.